spec/models/user_spec.rb

```diff
@@ -102,4 +101,27 @@
       expect(user_2).to_not be_valid
     end
   end
+
+  describe "methods" do
+    it 'can find item orders by merchant user with order id' do
+      regular_user_1 = create(:user)
+
+      merchant_shop_1 = create(:merchant, name: "Merchant Shop 1")
+        item_1 = merchant_shop_1.items.create!(attributes_for(:item, name: "Item 1" ))
+        item_2 = merchant_shop_1.items.create!(attributes_for(:item, name: "Item 2"))
+
+      order_1 = create(:order)
+        item_order_1 = regular_user_1.item_orders.create!(order: order_1, item: item_1, quantity: 2, price: item_1.price, user: regular_user_1)
+        item_order_2 = regular_user_1.item_orders.create!(order: order_1, item: item_2, quantity: 8, price: item_2.price, user: regular_user_1)
+
+      order_2 = create(:order)
+        item_order_4 = regular_user_1.item_orders.create(order: order_2, item: item_2, quantity: 18, price: item_2.price, user: regular_user_1)
+
+      merchant_admin = create(:user, role: 1, merchant: merchant_shop_1)
```

Comment:
Here too, do we need to have a merchant_admin (with role 2) or merchant_employee (with role 1) ?

Reply:
Updated and pushed up! Thanks